influxdb_connector/influxdb_unittests.py

```diff
@@ -11,18 +11,29 @@ class TestInfluxdb(unittest.TestCase):
     def setUp(self) -> None:
         self.args = []
         # read text file containing required arguments on each line
-        with open('arguments.txt') as f:
+        with open("arguments.txt") as f:
             self.args.append(f.readline())
-   
+
+
```

Comment:
Nice testcases to start with. 

Recommend adding more testcases say for bad query or bad csv name for example

Reply:
NoneCnvrgDatasetError() added as custom exception for bad query